### PR TITLE
Mondo master: ServerTests StaticFiles and Session

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by natemcmaster** [comment]
Can you also update the Directory.Build.props files to use the right url? This impacts sourcelink and symbols generation, so we should fix this when merging to new repos. We can go back and fix this later in release/2.1 and 2.2 which isn't impacted by this yet.

```diff
-    <RepositoryUrl>https://github.com/aspnet/AADIntegration</RepositoryUrl>
+    <RepositoryUrl>https://github.com/aspnet/AspNetCore</RepositoryUrl>
```

**Comment by Tratcher** [comment]
Are you running the tests? We're expecting issues with ServerTests similar to https://github.com/aspnet/AspNetCore-Internal/issues/1320#issuecomment-440336006

**Comment by ryanbrandenburg** [comment]
@natemcmaster: updated.
@Tratcher: We're letting the PR checks check for breaks for us, but if we know this is going to happen is someone working on a fix already?

**Comment by Tratcher** [comment]
They're still designing a fix.

File: build/dependencies.props [hunk 1]
```diff
@@ -210,6 +210,7 @@
     <MicrosoftVisualStudioTextUIPackageVersion>16.0.142-g25b7188c54</MicrosoftVisualStudioTextUIPackageVersion>
     <MicrosoftVisualStudioThreadingPackageVersion>15.8.168</MicrosoftVisualStudioThreadingPackageVersion>
     <MicrosoftWebAdministrationPackageVersion>11.1.0</MicrosoftWebAdministrationPackageVersion>
+    <MicrosoftVisualStudioWebBrowserLinkPackageVersion>2.2.0</MicrosoftVisualStudioWebBrowserLinkPackageVersion>
```
Comment (natemcmaster) [inline build/dependencies.props]: You can remove this line.

File: build/buildorder.props [hunk 2]
```diff
@@ -12,24 +12,24 @@
     <RepositoryBuildOrder Include="BasicMiddleware" Order="9" />
     <RepositoryBuildOrder Include="Antiforgery" Order="10" />
     <RepositoryBuildOrder Include="IISIntegration" Order="10" RootPath="$(RepositoryRoot)src\IISIntegration\" />
-    <RepositoryBuildOrder Include="StaticFiles" Order="11" />
-    <RepositoryBuildOrder Include="ResponseCaching" Order="11" />
-    <RepositoryBuildOrder Include="Session" Order="11" />
-    <RepositoryBuildOrder Include="ServerTests" Order="11" />
     <RepositoryBuildOrder Include="CORS" Order="12" />
+    <RepositoryBuildOrder Include="StaticFiles" Order="11" RootPath="$(RepositoryRoot)src\StaticFiles\" />
```
Comment (natemcmaster) [inline build/buildorder.props]: Can you restore the order of this file? MSBuild doesn't actually sort numerically, so things are building out of order now.

```
 10. Antiforgery, IISIntegration
 12. CORS, Routing, Diagnostics  
 11. StaticFiles, ResponseCaching, Session, ServerTests
```

**Review by natemcmaster — CHANGES_REQUESTED** [review]

**Comment by natemcmaster** [comment]
Also, let's resolved this first: https://github.com/aspnet/AspNetCore-Internal/issues/1411

**Comment by ryanbrandenburg** [comment]
:up::date:

**Review by natemcmaster — APPROVED** [review]

**Pull request merged**